lib/util.js

```diff
@@ -897,8 +901,12 @@ function formatTypedArray(ctx, value, recurseTimes, keys) {
   const maxLength = Math.min(Math.max(0, ctx.maxArrayLength), value.length);
   const remaining = value.length - maxLength;
   const output = new Array(maxLength + (remaining > 0 ? 1 : 0));
+  const elementFormatter =
+    types.isBigInt64Array(value) || types.isBigUint64Array(value) ?
```

Comment:
That would result in `elementFormatter` having the wrong value when the array is empty, although admittedly it wouldn't matter in that case because `elementFormatter` would never be called. I'll change it to use `typeof`.